assets/js/src/form_editor/components/form_title.jsx

```diff
@@ -2,15 +2,14 @@ import React, { useState } from 'react';
 import { useDispatch, useSelect } from '@wordpress/data';
 import classnames from 'classnames';
 import MailPoet from 'mailpoet';
-import Textarea from 'react-autosize-textarea';
```

Comment:
This was probably the only place where 'react-autosize-textarea' was used. I guess we can remove the package now.